### ISSUE TITLE
Decide whether to make conversations persist

### ISSUE THREAD

**Issue by lydgate** [Gryphire/wisperbot#39]
Looking at [this](https://github.com/python-telegram-bot/python-telegram-bot/wiki/Making-your-bot-persistent), we should decide whether it's important to save things like conversation state across bot reboots.

**Comment by lydgate** [comment]
See also this https://docs.python-telegram-bot.org/en/stable/telegram.ext.conversationhandler.html#conversationhandler

**Comment by Gryphire** [comment]
Yep, having thought about it a bit, I think it's important that we make sure conversations persist across potential (planned or unplanned) bot reboots. I believe this is not yet the case, right?

**Comment by lydgate** [comment]
Estimate 2 days

**Comment by lydgate** [comment]
See issue #68 for this.
